src/scope.ts

```diff
@@ -17,7 +18,7 @@ const SCOPE: { [x: string]: Function } = {
     return new bson.Binary(Buffer.from(d, 'base64'), t);
   },
   UUID: function(u: any) {
-    return new bson.Binary(Buffer.from(u.replace(/-/g, ''), 'hex'), 4);
+    return new bson.Binary(Buffer.from((u || randomUUID()).replace(/-/g, ''), 'hex'), 4);
```

Comment:
Do we want to handled `undefined`/`null` here only, and let `UUID(0)` etc. fail (as they should)?

```suggestion
    return new bson.Binary(Buffer.from((u ?? randomUUID()).replace(/-/g, ''), 'hex'), 4);
```